composer.json

```diff
@@ -33,10 +33,20 @@
             "module": "Laminas\\Diactoros"
         }
     },
+    "repositories": {
+        "http-factory": {
+            "type": "vcs",
+            "url": "https://github.com/ghostwriter/http-factory.git"
+        },
+        "integration-tests": {
+            "type": "vcs",
+            "url": "https://github.com/weierophinney/psr7-integration-tests"
+        }
+    },
```

Comment:
To be dropped before merge

Reply:
Yep - that's why this is in draft state. :smile:

Reply:
Figured: just making sure it isn't forgotten :+1: